com.github.makhber.Makhber.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.github.makhber.Makhber
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: makhber
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
Since you have socket=wayland, you should only have socket=fallback-x11 instead of socket=x11

Reply:
Does it have any side-effect on non-wayland environments?

Reply:
If there is no wayland, fallback-x11 is the fallback to X11.  so no side effect.
If there is wayland there is no x11 socket.